src/styles/components/table.scss

```diff
@@ -0,0 +1,64 @@
+@import "../core/settings";
+@import "../core/mixins";
+@import "../core/functions";
+
+$block: ns(table);
+
+/*!
+    .tn-table
+        .tn-table__row, tn-table__row--stripe | .is-active
+        .tn-table__cell
+*/
+
+// TABLE
+//     - not-striped
+
+
+
+.#{$block} {
+    //BASE
+    //set all BLOCK reset and baseline styles
+    @include reset;
+    border-collapse: collapse;
+    width: 100%;
+    background-color: color(light);
+
+    //VARIANTS
+    //each variant MUST extend base block
+    // &--modifier {
+    //     @extend .#{$block};
+    // }
```

Comment:
do we need to retain this commented code? in the past, we did not keep commented to the keep the files clean as you could recover the code from file history